README.rst

```diff
@@ -29,33 +41,244 @@ To evolve Figures to meet community needs, we are keeping in mind as principles
+
+As a top level key, add the following::
+
+	"ADDL_INSTALLED_APPS": [
+		"figures"
+	]
+
+We suggest adding the above immediately after ``ACTIVATION_EMAIL_SUPPORT_LINL`` so that it is in alphabetical order.
+
+In the FEATURES section, add ``"ENABLE_FIGURES": true``::
+
+	"FEATURES": {
+		... 
+		"ENABLE_FIGURES": true,
+		...
+	}
+
+
+5. Update your edx-platform
+
+You can update edx-platform from either the host or the VM. If you edit from the host, your edx-platform project is here::
+
+	<project>/edx-platform
+
+If you edit from within the VM, your edx-platform project is here::
+
+	/edx/app/edxapp/edx-platform
+
+
+Edit ``lms/urls.py`` to add the following to the bottom of the file::
+
+
+	if settings.FEATURES.get('ENABLE_FIGURES'):
+    	urlpatterns += (
+    		url(r'^figures/',
+    		    include('figures.urls', namespace='figures')),
+    	)
+
+
+Edit ``lms/envs/devstack.py`` to add the following to the bottom of the file::
+
+	from figures.settings import FIGURES
+
+
+6. Run migrations for Figures
+
+In the Vagrant VM, as user *edxapp*, navigate to the following directory::
+
+	/edx/app/edxapp/edx-platform
+
+And run the following::
+
+	./manage.py lms migrate figures
+
+
+7. Install Figures front-end dependencies
+
+As the *edxapp* user, navigate to the ``/edx/src/figures/frontend`` directory and run the following::
+
+	npm install
+
+This will install the `NPM <https://www.npmjs.com/>`_ dependencies.
+
+
+You should now have your devstack ready to run.
+
+
+Running Figures in Devstack
+===========================
+
+You will need two terminal windows open. One to start the LMS, the other to start the webpack development server for the Figures UI. In each, you should be user *edxapp*::
+
+1. Start the webpack development server::
+
+	cd /edx/src/figures/frontend
+	npm start
+
+This will start the Webpack development server on port 3000.
+
+
+2. Start the LMS::
+
+	cd /edx/app/edxapp/edx-platform
+
+	paver devstack lms
+
+
+3. Open the LMS in a browser on your host
+
+	a. Go to ``localhost:8000`` and log in as a staff or admin user
+	b. navigate to ``localhost:8000/figures/``
+
+
+The Figures main page should now be displayed.
+
+
+Production Installation
+-----------------------
+
+**NOTE: We're actively developing Figure toward production release this summer.** This means there are missing parts. Please see the notice_section_ section above. Also, these instructions are a work in progress (WIP). Your feedback is welcome so that we can improve the instructions**
 
-**NOTE: These instructions are a WIP and incomplete**
 
 This section describes installing Figures in Open edX "Ginkgo" release. This is the `open-release/ginkgo.master` branch of edX's `edx-platform <https://github.com/edx/edx-platform/tree/open-release/ginkgo.master>`_ LMS.
 
 For installing on Appsembler's `edx-platform fork <https://github.com/appsembler/edx-platform/tree/appsembler/ginkgo/master>`_ read **<TODO: Insert link to instructions doc>**
```

Comment:
Will this be in a separate PR?

Reply:
Yes. I created a trello card: https://trello.com/c/PSpGxXF6/445-1-update-production-installation-instructions-in-figures-readme

Reply:
To note, we've only run Figures in devstack. I need to add the production asset compilation script and test/refine installation on GCP.